pkg/download/automation/automation.go

```diff
@@ -101,6 +105,41 @@ func (d *Downloader) Download(projectName string, automationTypes ...config.Auto
 type NoopAutomationDownloader struct {
 }
 
+func createTemplateFromRawJSON(obj automationClient.Response) (t template.Template, extractedName string) {
+	configId := obj.Id
+
+	var data map[string]interface{}
+	err := json.Unmarshal(obj.Data, &data)
+	if err != nil {
+		log.Warn("Failed to sanitize downloaded JSON for config %v - template may need manual cleanup: %v", configId, err)
+		return template.NewDownloadTemplate(configId, configId, string(obj.Data)), configId
+	}
+
+	// remove properties not necessary for upload
+	delete(data, "id")
+	delete(data, "modificationInfo")
+	delete(data, "lastExecution")
+
+	// extract 'title' as name
+	configName := configId
+	if title, ok := data["title"]; ok {
+		configName = fmt.Sprintf("%v", title)
+		data["title"] = "{{.name}}"
```

Comment:
Shouldn't this happen anyway? If it's not there, should we still set it?

Reply:
won't change as discussed